package.json

```diff
@@ -2,12 +2,12 @@
   "name": "jquery",
   "title": "jQuery",
   "description": "JavaScript library for DOM operations",
-  "version": "3.7.1-pre",
+  "version": "3.7.0",
```

Comment:
I am not a fan of this change. With it, the version in package.json can no longer be universally trusted. Also, unless this was addressed somewhere else, Git builds will contain an incorrect version in the comment header and in the `jquery` property.

Reply:
Judging from the discussion at https://github.com/release-it/release-it/issues/946, it should be possible to keep our versioning strategy.

Reply:
> the version in package.json can no longer be universally trusted

It never really could with `-pre` either. It was often wrong about what version was coming next. `-pre` is a convention we invented that I don't think has a place anymore. The concern was that people would mistake some state of the repo with the exact state used to release the last version. But, the practice of leaving the version and continuing to develop has become so common I don't think anyone actually makes that mistake anymore. It's part of the language of maintaining semantically versioned repos.

In fact, because what we do is so foreign, it has gotten increasingly confusing. Keep in mind that changing to `-pre` obfuscates what the last version actually was. And the advantages of leaving the version alone have only been increasing. 

If `-pre` had caught on and become a standard, it might have been nice, but now we're the outliers.

> Git builds will contain an incorrect version in the comment header and in the jquery property.

Again, I don't the `-pre` version made sense either. As I explained in the issue description, git jQuery already has the SHA. All we'd need to do is remove the version from the header.

> Judging from the discussion at https://github.com/release-it/release-it/issues/946, it should be possible to keep our versioning strategy.

I'm not sure that issue is related. It's talking about alpha and beta releases, which actually work better when the version in the package.json doesn't have a custom suffix. release-it has a much easier time determining the next version (and give you alpha and beta versions options) when the package.json version is left alone, which I think helps illustrate my point about the strangeness of what we do.

Reply:
OK, I see your points about `package.json`.

I still feel a bit uneasy about the value of the `jquery` property; it's true we add the commit hash to the Git version but AFAIK that doesn't happen for local builds from commits. Maybe we could tweak the build process to always read the current Git hash (instead of relying on the environment variable) and always appending it on non-tags?

Reply:
I was actually thinking that as well. I'm also preparing a PR to migrate the build process off of grunt and I also like the idea of defaulting to adding the SHA.